products/BellHybrid/init_prof.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
```

Comment:
do we really need separate files for init for bell and pure here? :)

Reply:
Probably no. Why it wasn't added for the Bell in the first place? ;)

Reply:
Because I do not have a bell device for tests :joy: